### PR TITLE
refactor: Rename helpers for validation module

### PR DESCRIPTION
**Pull request by pankajagrawal16**
**Issue #, if available:**

## Description of changes:

**Checklist**

 

* [x] [Meet tenets criteria](https://awslabs.github.io/aws-lambda-powertools-java/#tenets)
* [x] Update tests
* [x] Update docs
* [x] PR title follows [conventional commit semantics]()

## Breaking change checklist

**RFC issue #**:

* [ ] Migration process documented
* [ ] Implement warnings (if it can live side by side)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by jeromevdl** [comment]
Maybe also rename ValidatorConfig to ValidationConfig ?

**Comment by pankajagrawal16** [comment]
> Maybe also rename ValidatorConfig to ValidationConfig ?

Done

File: powertools-validation/src/main/java/software/amazon/lambda/powertools/validation/ValidationUtils.java [hunk 1]
```diff
@@ -30,11 +30,14 @@
 /**
  * Validation utility, used to manually validate Json against Json Schema
  */
-public class Validator {
+public class ValidationUtils {
     private static final String CLASSPATH = "classpath:";
 
     private static final ConcurrentHashMap<String, JsonSchema> schemas = new ConcurrentHashMap<>();
 
+    private ValidationUtils() {
```
Comment (jeromevdl) [inline powertools-validation/src/main/java/software/amazon/lambda/powertools/validation/ValidationUtils.java:38]: 👍

**Review by jeromevdl — APPROVED** [review]
OK

**Review by stevehouel — APPROVED** [review]

**Pull request merged**